orchestrator/src/handlers/price/mod.rs

```diff
@@ -28,7 +28,10 @@ pub static IS_RUNNING: AtomicBool = AtomicBool::new(false);
 
 pub type ResponseResult = Result<Response, ErrorResponse>;
 
-pub async fn handler(notary_information: Arc<NotaryInformation>) {
+pub async fn handler(
+    notary_information: Arc<NotaryInformation>,
+    latest_valid_logs: Vec<EventLog>
```

Comment:
As we do not fetch any logs but instead pass them as a vector, please correct the debug messages to not confuse further logs investigation.

Reply:
Thanks on it